### PR TITLE
ADD documentation search on homepage

### PR DESCRIPTION
**Pull request by ndelangen**
It fetches content-lite.json and matches the provided string from a input-field
with all values on the page-object.

### CODE REVIEW

**Comment by ndelangen** [comment]
Seems like travis or dependency had connection problems causing travis-ci to mark the build as failure.

**Comment by ndelangen** [comment]
It'd be awesome to get some response on this?

**Comment by ashleygwilliams** [comment]
could you add some screenshots of what this looks like? thanks

File: public/styles/_homepage.styl [hunk 1]
```diff
@@ -24,4 +27,40 @@
...
+    margin 15px 0
+
+  h2
+    centeredColumn homepageWidth
+    margin-top 30px
+    margin-bottom 10px
+    &:after
+      color rgba(text-color, 0.4)
+
+.search
```
Comment (ashleygwilliams) [inline public/styles/_homepage.styl]: would you mind breaking this out into a `_search.styl`?

File: public/js/main.js [hunk 2]
```diff
@@ -14,4 +14,78 @@ $(function () {
         $('#npm-install-instructions').show()
       }
     })
+
+  var getContentLite = function (callback) {
```
Comment (ashleygwilliams) [inline public/js/main.js]: would you mind breaking this out into a `search.js`

**Comment by ashleygwilliams** [comment]
hey this is really awesome! thanks for contributing and thanks for your patience! a few nits about putting things into separate files- adding screenshots will help us deal with the likely styling nits! @gabra keep an eye out :)

**Comment by ndelangen** [comment]
https://youtu.be/kJxu7PcJEt8

**Comment by ndelangen** [comment]
I'm guessing I can just close this, because it doesn't look like npm is interested in getting contributions.

**Comment by ndelangen** [comment]
👋

**Pull request closed without merging**